src/App.tsx

```diff
@@ -1,16 +1,24 @@
-import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
+import './App.scss';
 
 export const App = () => {
+  const [newMovieList, setNewMovieList] = useState(moviesFromServer);
+
+  const onAdd = (newMovieObj: Movie) => {
```

Comment:
Don't need to add `obj` to the variable name. You can specify the type by TS
```suggestion
  const onAdd = (newMovie: Movie) => {
```